cg/meta/orders/api.py

```diff
@@ -57,7 +57,7 @@ def submit(self, project: OrderType, data: dict, ticket: dict) -> dict:
             # open and assign ticket to order
             try:
                 if self.osticket:
-                    message = self._create_new_ticket_message(data, ticket)
+                    message = self._create_new_ticket_message(data, ticket, project)
```

Comment:
Could you add variable names? It is tricky to see if wrong name is used

Reply:
fixed